workspaces/services/guacamole/porter.yaml

```diff
@@ -0,0 +1,92 @@
+name: tre-service-guacamole
+version: 0.1.2
+description: "An Azure TRE service for Guacamole"
+registry: azuretre
+dockerfile: Dockerfile.tmpl
+
+credentials:
+  - name: azure_tenant_id
+    env: ARM_TENANT_ID
+  - name: azure_subscription_id
+    env: ARM_SUBSCRIPTION_ID
+  - name: azure_client_id
+    env: ARM_CLIENT_ID
+  - name: azure_client_secret
+    env: ARM_CLIENT_SECRET
+
+parameters:
+  - name: workspace_id
+    type: string
+  - name: tre_id
+    type: string
+  - name: acr_name
+    type: string
+    description: "Name of the azure container registry"
+    env: ACR_NAME
+  - name: mgmt_resource_group_name
+    type: string
+    description: "Resource group containing the shared ACR"
+    env: MGMT_RESOURCE_GROUP_NAME
+  - name: guacamole_image_tag
```

Comment:
should this have a default - not sure the user is going to specify it?

Reply:
need to talk about the version numbers. I will set a placeholder with v0.0.1, we can decide what version we would like